code/modules/mob/living/say.dm

```diff
@@ -272,11 +289,13 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 			return NOPASS
 	return 0
 
-/mob/living/lingcheck() //Returns 1 if they are a changeling. Returns 2 if they are a changeling that can communicate through the hivemind
+/mob/living/lingcheck() //1 is ling w/ no hivemind. 2 is ling w/hivemind. 3 is ling victim being linked into hivemind. 

```

Comment:
But that's irrelevant here right? This is just the communications check, the Linglink.dm and Changeling.dm already ensure non-carbons don't have any way to be Ling'd or join the Hivemind.